.pipelines/upgrade-templates.yaml

```diff
@@ -0,0 +1,13 @@
+steps:    
+  - template: templates/get-latest-tag.yaml
```

Comment:
Add comment to make this clear that latest is the previously published tag

Reply:
Will do this in a follow up PR

Reply:
As a part of: https://github.com/Azure/osm-azure/issues/75